test/OpenTelemetry.Tests.Stress/Skeleton.cs

```diff
@@ -41,7 +41,7 @@ static Program()
         StressMeter.CreateObservableGauge("Process.VirtualMemorySize64", () => process.VirtualMemorySize64);
     }
 
-    public static void Stress(int concurrency = 0, int prometheusPort = 0)
+    public static void Stress(Action run, int concurrency = 0, int prometheusPort = 0)
```

Comment:
I guess Action would defeat the purpose of `[MethodImpl(MethodImplOptions.AggressiveInlining)] static void Run()`. What's the perf number (CPU cycles per loop) for [the naive test](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/test/OpenTelemetry.Tests.Stress)?

Reply:
You're right! The perf did go down with this change for the naive test. CPU Cycles/Loop went from 3 to 5 with this change.

![image](https://user-images.githubusercontent.com/66651184/155775072-ef81e9a0-873c-4946-a44c-09a66b623832.png)

Reply:
I guess we could close this PR and anytime someone wants to run Stress Tests for Histogram they can just update the `Run` method.

Reply:
Or add the metrics code but comment it out.